_includes/faq.scss

```diff
@@ -44,301 +44,103 @@
-    }
-}
-
-.faq-layout__contents {
-    flex: 0 0 200px;
+h2.fag-layout__title {
+    color: #154273;
+    background: #EFF7F9;
+    padding: 0.6em 20px;
+    margin-bottom: 0;
 }
 
 .faq-layout__contents__list {
-    @media (max-width: 739px) {
-        display: none;
-    }
-
-    li {
-        list-style: none;
-        
-        a {
-            display: block;
-            transition: var(--transition-out);
-            color: $color-RO_donkerblauw--interaction;
-            display: block;
-            margin-bottom: 19px;
-            font-size: 18px;
-            line-height: 20px;
-            padding-right: 10px;
-            border-left: 0px solid rgba(31, 82, 155, 0);
-
-            &.tabbable-tab--active {
-                border-left: 2px solid #1F529B;
-                padding-left: 8px;
-                color: #333;
-                transition: var(--transition-in);
-                font-weight: 600;
-            }
-        }
-    }
-}
-
-.faq-layout__tab-panels {
-    flex: 1 1;
-    max-width: 674px;
-}
-
-.faq-layout__tab-panels__updates-will-follow {
-    margin: 20px 0 0 0;
-    font-size: 14px;
-}
-
-// Panel
-
-.faq-tab-panel {
     position: relative;
+    border: 1px solid rgba(21, 66, 115, 0.5);
+    border-radius: 5px;
+    width: 100%;
+    margin-bottom: 20px;
+    overflow: hidden;
 
-    @media (min-width: 740px) {
-        .tabbable-js-interactive & {
-            display: none;
-            z-index: -1;
-            margin-top: -100px;
-            padding-top: 100px;
-            margin-bottom: -1px;
-            opacity: 0;
-            height: 1px;
-            overflow: hidden;
-
-            &.tabbable-tab-panel--active {
-                display: block;
-                z-index: auto;
-                margin-bottom: 0;
-                opacity: 1;
-                height: auto;
-                overflow: visible;
-                transition: opacity var(--transition-in);
-            }
-        }
-    }
-}
-
-.faq-tab-panel__title {
-    font-weight: 600;
-    color: $color-RO_donkerblauw;
-    font-size: 18px;
-    margin-top: 27px;
-    margin-bottom: 6px;
-    line-height: 20px;
-    
-    .tabbable-js-interactive & {
-        @media (min-width: 740px) {
-            position: absolute;
-            left: -9999px;
-        }
-    }
+    li:first-child .fag-item{
+        border-top: none;
+    } 
 }
 
-
-
-
-
-
-
-
-
-
-.faq details {
+.fag-item {
+    display: block;
     position: relative;
-    border: 1px solid #154273 /* RO blauw */;
-    border-radius: 5px;
+    color: #154273;
+    border-top: 1px solid rgba(21, 66, 115, 0.5);
+    padding: 0.8em 0.5em;
+    color: #154273;
+    font-size: 21px;
+    line-height: 22px;
     width: 100%;
-    margin-bottom: 10px;
     overflow: hidden;
 
     &:hover {
         background-color: rgba(0, 39, 131, 0.07) /* RO donker link2 */;
-        border-color: $color-RO_donkerblauw--interaction;
+        // border-color: $color-RO_donkerblauw--interaction;
```

Comment:
Deze mag er van mij helemaal uit, blijft toch in git history mocht het later weer nodig zijn.